lib/ansible/modules/windows/win_domain_computer.py

```diff
@@ -0,0 +1,122 @@
+    Set-ADComputer, Remove-ADComputer and Move-ADObject.
+options:
+  name:
+    description:
+      - >
+        Specifies the name of the object. This parameter sets the Name property
+        of the Active Directory object. The LDAP display name (ldapDisplayName)
+        of this property is name.
+    required: true
+  sam_account_name:
+    description:
+      - >
+        Specifies the Security Account Manager (SAM) account name of the user,
+        group, computer, or service account. The maximum length of the
+        description is 256 characters. To be compatible with older operating
+        systems, create a SAM account name that is 15 characters or less. This
+        parameter sets the SAMAccountName for an account object. The LDAP
+        display name (ldapDisplayName) for this property is sAMAccountName.
+        Note: If the SAMAccountName string provided does not end with a $, a $
+        will be appended if needed.
+    default: C(name)
+  enabled:
+    description:
+      - >
+        Specifies if an account is enabled. An enabled account requires a
+        password. This parameter sets the Enabled property for an account
+        object. This parameter also sets the ADS_UF_ACCOUNTDISABLE flag of the
+        Active Directory User Account Control (UAC) attribute.
+    default: true
+  ou:
+    description:
+      - >
+        Specifies the X.500 path of the Organizational Unit (OU) or container
+        where the new object is created. Required when I(state=present).
+  description:
+    description:
+      - >
+        Specifies a description of the object. This parameter sets the value
+        of the Description property for the object. The LDAP display name
+        (ldapDisplayName) for this property is description.
+    default: ''
+  dns_host_name:
+    description:
+      - >
+        Specifies the fully qualified domain name (FQDN) of the computer. This
+        parameter sets the DNSHostName property for a computer object. The LDAP
+        display name for this property is dNSHostName. Required when
+        I(state=present).
+  state:
+    description:
+      - >
+        Specify whether to add C(present) or remove C(absent) the specified
+        access rule.
+    required: no
+    choices:
+      - present
+      - absent
+    default: present
+notes:
+  - Enabled --check --diff options
+  - Verbose option displays module internal variables in "debug" branch.
+version_added: 2.6
+author: "Daniel Sánchez Fábregas"
+'''
+
+EXAMPLES = r'''
+  - name: Add linux computer to Active Directory OU using a windows machine
+    win_domain_computer:
+      name: one_linux_server.my_org.local
+      sam_account_name: linux_server
+      dns_host_name: one_linux_server.my_org.local
+      ou: "OU=servers,DC=my_org,DC=local"
+      description: Example of linux server
+      enabled: yes
+      state: present
+      delegate_to: my_windows_bridge.my_org.local
```

Comment:
Syntax error, `delegate_to:` should be at the task-level, not module-level.

Reply:
Fixed.
It was a copy paste + anonymize servers thing.